cypress/integration/02 - items.ts

```diff
@@ -0,0 +1,68 @@
+const testItem = {
+    name: "Google",
+    username: "example@google.com",
+    password: "somethingsecret",
+    url: "https://google.com",
+};
+
+describe("Items", () => {
+    it("can create an item without errors", () => {
+        cy.login();
+
+        // Click plus sign
+        cy.get("pl-app").find("pl-items").find("pl-items-list").find("pl-button:eq(2)").click({ force: true });
+
+        // Give the app some time to render the animations
+        cy.wait(100);
+
+        // Click create
+        cy.get("pl-app").find("pl-create-item-dialog").find("footer pl-button.primary").click({ force: true });
+
+        cy.url().should("include", "/items/");
+        cy.url().should("include", "/new");
+
+        // Give the app some (more) time to render the animations
+        cy.wait(100);
+
+        // Fill in form
+        cy.get("pl-app")
+            .find("pl-items")
+            .find("pl-item-view")
+            .find("pl-input#nameInput")
+            .find("input")
+            .type(testItem.name, { force: true });
+        cy.get("pl-app")
+            .find("pl-items")
+            .find("pl-item-view")
+            .find("pl-scroller")
+            .find("pl-list")
```

Comment:
I assume we need this long explicit chain because of shadow dom? If so, you can probably omit `pl-scroller` and `pl-list` here because their content is actually rendered into the "light dom" so should be able to query past their boundaries.

Reply:
Yes, it needs to be here because of the shadow dom. I can try removing these and seeing if that works, if you want, but I honestly think this makes reading (and potentially updating) the test easier.